include/s3select.h

```diff
@@ -2184,6 +2188,10 @@ class base_s3object
   unsigned long m_processed_rows;
   size_t m_returned_bytes_size;
   std::function<void(const char*)> fp_ext_debug_mesg;//dispache debug message into external system
+  std::vector<std::string> m_projection_keys{};
+  std::vector<std::string> m_alias_keys{};
```

Comment:
the `m_alias_key` is used only in this function, there is not need for a data-member. 
it should be a scope variable.